pkg/codegen/schema/schema.go

```diff
@@ -1831,12 +1834,15 @@ const (
 )
 
 // Validate an individual name token.
-func (spec *PackageSpec) validateTypeToken(section, token string) hcl.Diagnostics {
+func (spec *PackageSpec) validateTypeToken(validPrefixes map[string]bool, section, token string) hcl.Diagnostics {
```

Comment:
Mind using the same naming throughout for improved readability/understandability?

```suggestion
func (spec *PackageSpec) validateTypeToken(allowedPackageNames map[string]bool, section, token string) hcl.Diagnostics {
```

Reply:
Good point